pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -121,6 +121,10 @@ func Pod(source *v1alpha1.CatalogSource, name string, image string, labels map[s
 						InitialDelaySeconds: livenessDelay,
 					},
 					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							v1.ResourceCPU:    resource.MustParse("100m"),
+							v1.ResourceMemory: resource.MustParse("100Mi"),
+						},
```

Comment:
This is the only part of the PR that I'm weary about given that Catalogs should continuously grow as more operators are created.

Reply:
@awgreene @Denton24646 do we have an idea of how these resources scale on an average AWS cluster as catalogs get bigger? could be that we need to set some reasonable limits anyway, and we probably can't just say "Catalogs can be of any size" with the current memory footprint as it scales. or maybe it handles scale well given normal pod resource limits? either way we should probably have some idea what those numbers look like before we merge this.